sdk/ai/ai-projects/eslint.config.mjs

```diff
@@ -0,0 +1,17 @@
+import azsdkEslint from "@azure/eslint-plugin-azure-sdk";
+
+export default [
+  ...azsdkEslint.configs.recommended,
+  {
+    rules: {
+      "@azure/azure-sdk/ts-modules-only-named": "warn",
```

Comment:
Please provide justification for changing the default rules for each rule, why were these needed?

Reply:
I don't remember this was added explicitly, generated as part of initial setup.